app/main.py

```diff
@@ -1,3 +1,73 @@
+from __future__ import annotations
+from typing import Any
+
+
+# solution
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: Any) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Any) -> int:
```

Comment:
The return type hint for the `__add__` method should be `Distance` instead of `int` since it returns a `Distance` object.